src/arithmetic/numeric_funcs/numeric_funcs.c

```diff
@@ -152,22 +131,27 @@ void Register_NumericFuncs() {
 
 	types = array_new(SIType, 1);
 	types = array_append(types, (SI_NUMERIC | T_STRING | T_ARRAY | T_BOOL | T_NULL));
-	func_desc = AR_FuncDescNew("add", AR_ADD, 2, VAR_ARG_LEN, types, true);
+	func_desc = AR_FuncDescNew("add", AR_ADD, 2, 2, types, true);
 	AR_RegFunc(func_desc);
 
 	types = array_new(SIType, 1);
 	types = array_append(types, (SI_NUMERIC | T_NULL));
-	func_desc = AR_FuncDescNew("sub", AR_SUB, 2, VAR_ARG_LEN, types, true);
+	func_desc = AR_FuncDescNew("sub", AR_SUB, 2, 2, types, true);
 	AR_RegFunc(func_desc);
 
 	types = array_new(SIType, 1);
 	types = array_append(types, (SI_NUMERIC | T_NULL));
-	func_desc = AR_FuncDescNew("mul", AR_MUL, 2, VAR_ARG_LEN, types, true);
+	func_desc = AR_FuncDescNew("mul", AR_MUL, 2, 2, types, true);
```

Comment:
Are we sure Arithmetic expression aren't "compressed" to include multiple arguments under the same operation, e.g. 1+2+3 ?